config/application.rb

```diff
@@ -19,7 +19,7 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
-
+    config.i18n.available_locales = [:en, :sv]
```

Comment:
had an issue where the gem carrierwaves master git branch added a ton of locales, solved it by doing this for now.